proto/stat_var.proto

```diff
@@ -232,4 +232,22 @@ message GetStatVarSummaryRequest {
 }
 message GetStatVarSummaryResponse {
   map<string, StatVarSummary> stat_var_summary = 1;
+}
+
+message GetStatVarMatchRequest {
+  // The property value pair. Ex
+  // {gender: Male}, {statType: measuredValue}
+  // To improve this, can explicitly specify "statType", "measuredProperty",
+  // and constraint properties. Then the match could have must-match terms and
+  // optional match terms that have weights. That would greatly reduce the
+  // number of returned match stat vars.
```

Comment:
@nopper Let me know if this makes sense

Reply:
Just to clarify, at the moment `statType` and `measuredProperty` will be ignored? Only constraint properties work?